code/code/misc/combat.cc

```diff
@@ -3853,13 +3853,8 @@ int TBeing::oneHit(TBeing *vict, primaryTypeT isprimary, TThing *weapon, int mod
   } else {
     if (vict->isPc() && vict->desc) {
       if (canAttack(isprimary)) {
-        if (victimCanAttack) {
-          vict->learnFromDoingUnusual(LEARN_UNUSUAL_NORM_LEARN, SKILL_DEFENSE, (200 - (2 * myLevel)));
-          vict->learnFromDoingUnusual(LEARN_UNUSUAL_NORM_LEARN, SKILL_ADVANCED_DEFENSE, (200 - (2 * myLevel)));
-        } else {
-          vict->learnFromDoingUnusual(LEARN_UNUSUAL_NORM_LEARN, SKILL_DEFENSE, (300 - (2 * myLevel)));
-          vict->learnFromDoingUnusual(LEARN_UNUSUAL_NORM_LEARN, SKILL_ADVANCED_DEFENSE, (300 - (2 * myLevel)));
-        }
+        vict->learnFromDoingUnusual(LEARN_UNUSUAL_NORM_LEARN, SKILL_DEFENSE, (140 - (2 * myLevel)));
+        vict->learnFromDoingUnusual(LEARN_UNUSUAL_NORM_LEARN, SKILL_ADVANCED_DEFENSE, (140 - (2 * myLevel)));
```

Comment:
What's the difference between this section and the section at line 3748? Can't quite wrap my head around what this block is for.

Reply:
Hrmm. Yeah It looks confusing. I removed the bit here about providing a greater chance to hone if you canAttack() because it seemed like it didn't add much value. I understand the idea behind it but in the interest of making honing it easier I just removed it.

Reply:
Sorry, I meant to "approve" it after writing this and forgot